src/locales/en/translation.json

```diff
@@ -1617,14 +1611,15 @@
       }
     },
     "cryptocurrencyPrices": {
-      "title": "CRYPTOCURRENCY PRICES BY 24H CHANGE",
+      "title": "Tokens Listed On Origins",
```

Comment:
![UI issue 4](https://user-images.githubusercontent.com/56412037/144449822-239e65dd-b09d-4d53-a25c-adddfff9876f.JPG)
In UI/UX, this table title is different. It must be "Tokens Listed On Sovryn Origins"